test/Trip-test.js

```diff
@@ -0,0 +1,203 @@
+import { expect } from "chai";
+import Trip from "../src/Trip";
+
+describe("Trip", () => {
```

Comment:
Would suggest reducing number of assertions in this test suite. For tests regarding existence of properties on the Trip class, testing one instance is sufficient. If there is logic that is applied in the constructor, then you would need different instances.